test/config/utility.h

```diff
@@ -315,6 +315,11 @@ class ConfigHelper {
       const envoy::extensions::filters::network::http_connection_manager::v3::LocalReplyConfig&
           config);
 
+  // Adjust the upstream route with larger timeout if running tsan. This is the duration between
+  // whole request being processed and whole response received.
+  static void adjustUpstreamTimeoutForTsan(
```

Comment:
Can you file a tracking bug for this one?  I'm concerned about QUIC being slower, both on tsan and not and I'd like us to look into it at some point.